modules/update-service-configure-cvo.adoc

```diff
@@ -0,0 +1,39 @@
+[id="update-service-configure-cvo_{context}"]
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+* The Openshift Update Service graph-data container image has already been created and pushed to a repository accessible to the OpenShift Update Service.
+* The current release and update target releases have already been mirrored to a locally accessible registry.
+* The OpenShift Update Service application has already been created.
+
+.Procedure
+
+. Set the OpenShift Update Service target namespace, for example, `openshift-update-service`:
++
+[source,terminal]
+----
+$ NAMESPACE=openshift-update-service
```

Comment:
It's not a big deal to do it this way, but setting these variables in individual steps seems like it's making the procedure a lot longer than it needs to be. Is there any reason to do it this way vs. showing a YAML code block and adding a replaceable value / callout to the lines that they need to set themselves? And then doing an `oc create -f <file>` after?

For example: https://docs.openshift.com/container-platform/4.7/operators/admin/olm-restricted-networks.html#olm-creating-catalog-from-index_olm-restricted-networks

Reply:
I didn't address this comment in the new PR, so calling it out in case you agree, @jottofar. If so, please open a new PR and we'll get this updated.